src/niivue/index.ts

```diff
@@ -10160,6 +10160,27 @@ export class Niivue {
         this.draw2D([0, 0, 0, 0], 2)
       } else {
         // sliceTypeMultiplanar
+        let showRender = false
+        // this.opts.multiplanarForceRender is deprecated but was boolean.
+        // We now need to check if it is true. If so, then the user may not know
+        // about the new multiplanarShowRender option, so we need to show the render.
+        if (this.opts.multiplanarForceRender) {
+          showRender = true
+          // warn the user that the option is deprecated
+          log.warn(
+            'multiplanarForceRender is deprecated. Use multiplanarShowRender instead. Possible values are: "always", "auto", "never".'
+          )
+        } else {
+          //  check the now preferred multiplanarShowRender option.
+          // if the value is always, then show the render.
+          if (this.opts.multiplanarShowRender === 'always') {
+            showRender = true
```

Comment:
This should be renamed with "is", like isShowingRender to clarify that this is a boolean.